webpaste.el

```diff
@@ -77,9 +78,9 @@ each run.")
                                         (when data
```

Comment:
Would be niced if you changed this function as well to cl-function when changing all the others :)